packages/vuetify/src/components/VList/VListItem.tsx

```diff
@@ -220,6 +220,7 @@ export const VListItem = genericComponent<VListItemSlots>()({
           ]}
           href={ link.href.value }
           tabindex={ isClickable.value ? (list ? -2 : 0) : undefined }
+          title={ slots.title?.({ title: props.title }) ?? props.title }
```

Comment:
I've never seen a slot used like this before. It also means you'd have to update the available slots for this component. I think it's safe for this to simply equal props.title.